docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -0,0 +1,70 @@
+= Getting Started
+
+This page describes how to order a new managed Redis instance and consume it in an application.
+
+Target Audience::
+Service Consumer (Developer)
+
+== Prerequisites
+
+* `kubectl`
+
+== Steps
+
+. Create a new namespace `my-redis`.
```

Comment:
This leaves the impression that the instance will be running in `my-redis`. Maybe rename to `my-project`?